Spanner/src/Batch/BatchSnapshot.php

```diff
@@ -185,6 +188,9 @@ public function partitionRead($table, KeySet $keySet, array $columns, array $opt
      *           the type should be given as an array, where the first element
      *           is `Database::TYPE_ARRAY` and the second element is the
      *           array type, for instance `[Database::TYPE_ARRAY, Database::TYPE_INT64]`.
+     *     @type bool $dataBoostEnabled This is an optional parameter which can
+     *           be used for partition read and query to execute the request via
+     *           spanner independent compute resources.
```

Comment:
It's already documented that its Configuration Options. So, we can keep this message crisp here.

```suggestion
     *     @type bool $dataBoostEnabled Execute the request via spanner independent
     *           compute resources.
```

Reply:
Pls also check whether this message best represents the intent of this flag.

Reply:
The message does represents this in its best interest because I wrote it after taking reference from [here](https://github.com/googleapis/google-cloud-php/blob/a66e6b855de420ed7975dfc17799327547ba13e1/Spanner/src/V1/ReadRequest.php)

Reply:
Which configuration options?🤔 I'm a little confused.

Reply:
Understood, the reference seems fine.